[1,"Error occurred during cleanup: Error: The process cannot access the file because it is being used by another process. (os error 32): remove 'C:\\Users\\richa\\AppData\\Local\\Temp\\quarto-sessionc42bf8f71ae32b0d'\n"]
